logfields/common.go

```diff
@@ -0,0 +1,25 @@
+package logfields
```

Comment:
It might be good to have some function for each const. Since one problem we have is one people will use different types for a name and kibana doesn't let you filter on it because one is a string and one is an int.

```go
type Logger zerolog.Logger

func (Logger l) WithVehicleTokenId(tokenId uint) Logger {...}
```

Reply:
yes, I was seeing that we have a couple data type mismatch errors in kibana (with number types), this looks like a nice solution.